walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -35,8 +36,7 @@
       return answer;
     }
 
-    // build in O(PlogP) so contains() run in O(logP) instead of O(P), where P is the max number of Attendees per Event
-    TreeSet<String> attendees = new TreeSet(request.getAttendees());
+    TreeSet<String> attendees = new HashSet(request.getAttendees());
```

Comment:
Typo, assigning HashSet to TreeSet, probably a compile error.

Reply:
My mistake! Overlooked it! Will change, thanks!